terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -0,0 +1,29 @@
+environment = "live"
```

Comment:
Since this service is not live, in my opinion we should not add variable for live env.